subiquity/ui/views/filesystem/partition.py

```diff
@@ -161,15 +161,27 @@ def valid_char(self, ch):
 
 
 class PartitionForm(Form):
-    def __init__(self, model, max_size, initial, lvm_names, device, alignment):
+    def __init__(
+        self,
+        model,
+        max_size,
+        initial,
+        lvm_names,
+        device,
+        alignment,
+        remote_storage: bool,
+    ):
         self.model = model
         self.device = device
         self.existing_fs_type = None
+        self.remote_storage: bool = remote_storage
         if device:
             ofstype = device.original_fstype()
             if ofstype:
                 self.existing_fs_type = ofstype
         initial_path = initial.get("mount")
+        if not initial_path and remote_storage:
+            initial["mount"] = "/home"
```

Comment:
Probably shouldn't do this if there is already a mount for /home?

Reply:
Yeah, it's not ideal. But since we show an error message in the form when a partition already exists, I do think that it's not that bad.

![2024-01-26T14:24:13,147115123+01:00](https://github.com/canonical/subiquity/assets/4038023/f6127fe1-b18b-4cde-859e-62044be47220)

Reply:
I guess that's OK